test/functional/apps/dashboard_elements/controls/options_list/options_list_allow_expensive_queries_off.ts

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { OPTIONS_LIST_CONTROL } from '@kbn/controls-plugin/common';
+import expect from '@kbn/expect';
+
+import { OPTIONS_LIST_ANIMAL_SOUND_SUGGESTIONS } from '../../../../page_objects/dashboard_page_controls';
+import { FtrProviderContext } from '../../../../ftr_provider_context';
+
+export default function ({ getService, getPageObjects }: FtrProviderContext) {
+  const browser = getService('browser');
+
+  const { dashboardControls, timePicker, console, common, dashboard, header } = getPageObjects([
+    'dashboardControls',
+    'timePicker',
+    'dashboard',
+    'console',
+    'common',
+    'header',
+  ]);
+
+  const setAllowExpensiveQueries = async (value: boolean) => {
+    await common.navigateToApp('console');
+    await console.closeHelpIfExists();
+    await console.clearTextArea();
+    await console.enterRequest(
+      '\nPUT _cluster/settings\n{"transient": {"search.allow_expensive_queries": ' + value + '}}'
+    );
+    await console.clickPlay();
+    await header.waitUntilLoadingHasFinished();
+    await browser.refresh();
+  };
+
+  describe('Allow expensive queries setting is off', () => {
```

Comment:
The expensive queries are most important, since it is assumed that `allow_expensive_queries` is **usually** true/undefined - so, these "cheap" tests are primarily a smoke test to ensure that nothing completely breaks when this setting is `false`.